tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/APIView/APIViewAuthTool.cs

```diff
@@ -0,0 +1,105 @@
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.ComponentModel;
+using System.Text.Json;
+using Azure.Sdk.Tools.Cli.Commands;
+using Azure.Sdk.Tools.Cli.Models;
+using Azure.Sdk.Tools.Cli.Models.APIView;
+using Azure.Sdk.Tools.Cli.Models.Responses;
+using Azure.Sdk.Tools.Cli.Services;
+using ModelContextProtocol.Server;
+
+namespace Azure.Sdk.Tools.Cli.Tools.APIView;
+
+[McpServerToolType]
+[Description("APIView authentication operations including status checks and guidance")]
+public class APIViewAuthTool : MCPMultiCommandTool
```

Comment:
I would prefer to remove this class altogether and return auth guidance as response in case of auth failure from actual tool calls. Please see below comment for more details.